clumper/clump.py

```diff
@@ -66,45 +66,36 @@ def read_json(cls, path):
     @classmethod
     def read_csv(cls, path, delimiter=",", fieldnames=None, nrows=None):
         """
-        Reads in a csv file. Can also read files from url.
-
-        Parameters
-        ----------
-        path : filename or url
-        delimiter: must be a single character. `,` is the default.
-        fieldnames: If `None`, the first row of the csv is assumed as the header.
-        nrows: Number of rows to read in. Useful when reading large files. If `None`, all rows are read.
+        Reads in a csv file. Can also read files from url. The delimiter should be a single character; the default is `,`.
```

Comment:
Best to look at the docstring for `group_by` and use that as an example. The previous comment was to refer to `Arguments:` instead of `Parameters.

Reply:
@koaning , my bad. I misunderstood your earlier comment. Restored previous information and replaced `Parameters` with `Arguments`. Also, I looked at the groupby doc, and there are no arguments/parameters listed there - kindly point me in the right direction.